sp/src/game/server/hl2/hl2_player.cpp

```diff
@@ -1760,7 +1760,12 @@ void CHL2_Player::StartAutoSprint()
 //-----------------------------------------------------------------------------
 void CHL2_Player::StartSprinting( void )
 {
+#ifdef EZ2
+	// Ignore when infinite sprint is active
+	if( sv_infinite_sprint_power.GetInt() != 2 && m_HL2Local.m_flSuitPower < 10 )
```

Comment:
I think maybe we should enumerate this just so that it's clear that 2 means infinite sprint while 1 means infinite aux power.

Maybe like SPRINT_POWER_ALL and SPRINT_POWER_SPRINTONLY or something.

Reply:
Where do you think this enum should be located?

Reply:
Since the usage is limited to only hl2_player.cpp, I think it would be fine to define it here. The important thing is that we're not just cryptically using 1 and 2 as different convar states without any explanation to what the expected behavior is